pylint/test/functional/no_else_raise.py

```diff
@@ -20,7 +20,7 @@ def foo2(x, y, w, z):
         raise Exception(w)
     else:
         c = 3
-        raise Exception(z)
+        return z
```

Comment:
I think we should not modify this here, unless this was a bug which was fixed by this PR (but does not seem to be the case here). If we need to add mixed testing, we should add separate test case.